townhall/myapi/models.py

```diff
@@ -35,4 +37,25 @@ class Organization(models.Model):
     email = models.CharField(max_length=254)
 
     def __str__(self):
-        return self.name
\ No newline at end of file
+        return self.name
+    
+
+class Post(models.Model):
+    id = models.AutoField(primary_key=True)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    content = models.TextField()
+    created_at = models.DateTimeField(default=timezone.now)
+
+    def __str__(self):
+        return self.id
```

Comment:
The str method should return a string, self.id is an integer. 
consider converting it to a string: `return str(self.id)`